hr_timesheet_retroactive_cost/wizards/hr_employee_timesheet_cost_wizard_views.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2024 Moduon Team S.L. <info@moduon.team>
+     License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+    <record id="hr_employee_timesheet_cost_wizard_view_form" model="ir.ui.view">
+        <field name="name">hr_employee_timesheet_cost_wizard.view.form</field>
+        <field name="model">hr.employee.timesheet.cost.wizard</field>
+        <field name="arch" type="xml">
+            <form string="Update Timesheet Cost">
+                <sheet>
+                    <group>
+                        <group>
+                            <field name="timesheet_cost" string="Cost" />
+                        </group>
+                        <group>
+                            <field
+                                name="retroactive_cost_date"
+                                string="From Date"
+                                required="1"
```

Comment:
**issue(dry):** if you follow the suggestion above and set `required=True` in the model, then you would be able to remove it from here:

```suggestion
```